content/starter-configs/tutorials/drupal-8.md

```diff
@@ -116,9 +114,9 @@ services:
         # files from another publicly-accessible Drupal site instead of
         # syncing the entire files directory into the Tugboat Preview.
         # This results in smaller previews and reduces the build time.
-        - drush -r "${DOCROOT}" pm-download stage_file_proxy
-        - drush -r "${DOCROOT}" pm-enable --yes stage_file_proxy
-        - drush -r "${DOCROOT}" variable-set stage_file_proxy_origin
+        - composer require --dev drupal/stage_file_proxy
+        - drush pm:enable --yes stage_file_proxy
```

Comment:
drush doesn't need to know the root of the drupal installation anymore? If that's the case, neat!

Reply:
Correct, it can find it by reading the composer.json.